arangod/Cache/PlainBucket.cpp

```diff
@@ -155,36 +185,50 @@ void PlainBucket::clear() noexcept {
   TRI_ASSERT(isLocked());
   _state.clear();  // "clear" will keep the lock!
 
-  for (std::size_t i = 0; i < slotsData; ++i) {
+  for (std::size_t i = 0; i < kSlotsData; ++i) {
     _cachedHashes[i] = 0;
     _cachedData[i] = nullptr;
   }
+  _slotsUsed = 0;
+  checkInvariants();
 
   _state.unlock();
 }
 
-void PlainBucket::moveSlot(std::size_t slot, bool moveToFront) noexcept {
+void PlainBucket::moveSlot(std::size_t slot) noexcept {
```

Comment:
Since this now always moves to front, can we actually call the function `moveSlotToFront`?